fall-2020/navigation.py

```diff
@@ -66,20 +66,53 @@ def __init__(self, GPS, steering):
 
     def set_desired_position(self, desired_position):
         """ Sets the desired position the TBM will attempt to move to.
-        
+
         Note: assume the user will pass in the desired_position parameter when using
         the interface 
         """
-        pass
+        self.desired_position = desired_position
+        return 0
 
     
     def update_current_position(self):
         """ Updates the current position of the TBM """
-        pass
+        self.current_position = self.GPS.getPos()
+        return 0
 
 
     def navigate(self):
-        #test pass
+        #tuple to store where to travel
```

Comment:
It is good style practice to start each comment with a space to increase readability.

For example: 
``` # Like this``` instead of 
``` #Like this```